README.md

```diff
@@ -29,39 +54,35 @@ points in time.
 This extension hooks into the SchemaTool generation process so that it will automatically
 create the necessary DDL statements for your audited entities.
 
-## Installation (In Symfony2 Application)
+## Installation
 
 ### Installing the bundle
 
-Simply run assuming you have installed composer.phar or composer binary:
+Simply run assuming you have composer:
 
 ``` bash
-$ php composer.phar require simplethings/entity-audit-bundle
+$ composer require sonata-project/entity-audit-bundle
 ```
 
 ### Enable the bundle
 
 Finally, enable the bundle in the kernel:
 
 ``` php
-// app/AppKernel.php
+// config/bundles.php
 
-public function registerBundles()
-{
-    $bundles = array(
-        //...
-        new SimpleThings\EntityAudit\SimpleThingsEntityAuditBundle(),
-        //...
-    );
-    return $bundles;
-}
+return [
+    //...
+    SimpleThings\EntityAudit\SimpleThingsEntityAuditBundle::class => ['all' => true],
+    //...
+];
 ```
 
 ### Configuration
 
-Load extension "simple_things_entity_audit" and specify the audited entities (yes, that ugly for now!)
+Load extension "simple_things_entity_audit" and specify the audited entities
 
-##### app/config/config.yml
+##### config/packages/entity_audit.yml
```

Comment:
IMO, we must place these paths as a comment inside the code snippet, like in the previous snippet for `config/bundles.php`. This way, we will be consistent with the rest of [our packages](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/docs/reference/configuration.rst#configuration-1).

Reply:
Done